src/main/java/sirius/biz/tenants/metrics/TenantMetricComputer.java

```diff
@@ -27,6 +37,8 @@
  *  <li><tt>num-active-users</tt>: Total number of <tt>active</tt> users.</li>
  *  <li><tt>avg-activity</tt>: Average activity level of all active users.</li>
  *  <li><tt>avg-education-level</tt>: Average education level level of all active users.</li>
+ *  <li><tt>num-processes</tt></tt>: The number of started processes.</li>
```

Comment:
Sind die Changes in der Datei beabsichtigt?

Reply:
Which changes? :-P